test/unit/foreman/renderer/snapshots/ProvisioningTemplate/finish/Kickstart default finish.snap.txt

```diff
@@ -83,7 +83,6 @@ ssldir = \$vardir/ssl
 [agent]
 pluginsync      = true
 report          = true
-ca_server       = 
```

Comment:
It would be nice to have some test where the data is set. Or testcases for the macros.

Reply:
Maybe tests for the macros, but altering the snapshot fixtures is quite out of scope of this PR, given we still have static set on wich we generate snapshots.